CK3toEU4/Data_Files/configurables/culture_map.txt

```diff
@@ -230,10 +230,10 @@ link = { eu4 = severian ck3 = severian tech = eastern gfx = easterngfx }
 
 # West Slavic
 
-link = { eu4 = pommeranian ck3 = pommeranian tech = western gfx = westerngfx }
+link = { eu4 = old_pommeranian ck3 = pommeranian tech = western gfx = westerngfx }
```

Comment:
add pommeranian culture in germanic block as an offshoot of german in vanilla pommeranian areas:
provinces 46, 2996, 4749, 2993, 4774
vorpomern area
hinterpommern area

province 45 from mecklenburg area to lower saxon
province 4786 from mecklenburg area to saxon
province 49 from neumark area to saxom
province 2969 from neumark area to saxon

drop neumark, vonpommern, hinterpommern areas from prussian